general/update_fars/02_combine_csv.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+import os
+import psycopg2
+from psycopg2 import sql
+from dotenv import load_dotenv
+from zipfile import ZipFile
+from sqlalchemy import create_engine
+from io import StringIO
+
+# get environment variables
+load_dotenv("rds_conn_vars.env")
```

Comment:
Is there documentation somewhere that says how to create this? If not, should we add it here?